lib/html_parser.dart

```diff
@@ -96,14 +100,22 @@ class HtmlParser {
       }
       switch (node.localName) {
         case "a":
-          return DefaultTextStyle.merge(
-            child: Wrap(
-              children: _parseNodeList(node.nodes),
-            ),
-            style: const TextStyle(
-                decoration: TextDecoration.underline,
-                color: Colors.blueAccent,
-                decorationColor: Colors.blueAccent),
+          return GestureDetector(
+            child: DefaultTextStyle.merge(
+              child: Wrap(
+                children: _parseNodeList(node.nodes),
+              ),
+              style: const TextStyle(
+                  decoration: TextDecoration.underline,
+                  color: Colors.blueAccent,
+                  decorationColor: Colors.blueAccent),
+            ),
+            onTap: () {
+              if (node.attributes.containsKey('href')) {
```

Comment:
Can you add a check for onLinkTap being null?

`if (node.attributes.containsKey('href') && onLinkTap != null) {`